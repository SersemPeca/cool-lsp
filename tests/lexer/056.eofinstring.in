"EOF in multi-line string \
string continues; EOF here: