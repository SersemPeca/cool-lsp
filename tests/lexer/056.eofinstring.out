#2 ERROR: Unterminated string at EOF
